### DISCUSSION TITLE
mrv2 v1.0.2

### DISCUSSION THREAD

**Discussion by ggarra13** [ggarra13/mrv2#182] [Announcements]
v1.0.2
======

This is mainly a solidification of NDI® support.

- Added a Gigabytes setting to NDI® Panel to allow reproducing 4K and higher movie files with synced audio.
- Fixed NDI® streams playing with audio sometimes hanging at start up.
- Added a "No Source" to disconnect from NDI®.
- Made "Remote Connection" not be a valid NDI® connection.
- Fixed closing and reopening panel not showing the selected source.
- Fixed hang ups when switching sources in NDI® panel with videos with audio.  Videos without audio were fine.
- Removed references to NDI/Source Index interal Setting.
- Improved startup times of NDI® Sources with Audio.

<hr /><em>This discussion was created from the release <a href='https://github.com/ggarra13/mrv2/releases/tag/v1.0.2'>mrv2 v1.0.2</a>.</em>

**Comment by srinivasdotc** [comment]
It is crashing on Mac. sonoma OS.
error is,
Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: @rpath/libndi.dylib

**Comment by ggarra13** [comment]
Sorry for that.  I missed to include the NDI library.  Try the following installer:

https://mega.nz/file/rDRXlZrC#AGWfD6jLtWsGa6JePaiCxNI82gdVMe84EKRdO_hLXNQ

**Reply by srinivasdotc** [reply]
It is working now. Thank you.
